hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/IncrementalContainerReportHandler.java

```diff
@@ -108,9 +108,14 @@ public void onMessage(final IncrementalContainerReportFromDatanode report,
         } catch (ContainerReplicaNotFoundException e) {
           LOG.warn("Container {} replica not found!",
               replicaProto.getContainerID());
-        } catch (NotLeaderException nle) {
-          LOG.warn("Failed to process " + replicaProto.getState()
-              + " Container " + id + " due to " + nle);
+        } catch (SCMException ex) {
+          if (ex.getResult() == SCMException.ResultCodes.SCM_NOT_LEADER) {
+            LOG.warn("Failed to process " + replicaProto.getState()
```

Comment:
Can we change this to the log placeholder syntax?

```
LOG.warn("Failed to process {} container {}", replicaProto.getState(), id, ex);
```

I know it was like this before you made this change, but we may as well tidy it up as part of this change.

Reply:
Done